### PR TITLE
[Review] Expose Keyboard + Mouse Properly

### PR DESCRIPTION
**Pull request by ziahamza**

### CODE REVIEW

**Review by prayogoa — APPROVED** [review]

File: lib/webrtcstreamer.js [hunk 1]
```diff
@@ -188,21 +188,55 @@ export default class WebRtcStreamer {
...
+      if (this.mouse) {
+        this.mouse.ungrab();
+      }
+    }
+  };
+
+  keyboardGrabbed = false;
+  keyboard = null;
+  _keyboard = {
+    grab = () => {
```
Comment (ronaldpedagat) [inline lib/webrtcstreamer.js:211]: grab/ungrab: ?

File: lib/webrtcstreamer.js [hunk 2]
```diff
@@ -188,21 +188,55 @@ export default class WebRtcStreamer {
     }
   }
 
+  mouseGrabbed = false;
+  mouse = null;
+  _mouse = {
+    grab = () => {
```
Comment (ronaldpedagat) [inline lib/webrtcstreamer.js:194]: grab/ungrab: ?

Comment (ziahamza) [reply lib/webrtcstreamer.js:194]: @racpa I dont understand. both are implemented here.

File: lib/webrtcstreamer.js (hunk 1 above)
Comment (ziahamza) [reply lib/webrtcstreamer.js:211]: not sure what you mean there @racpa .

**Pull request merged**